### ISSUE TITLE
eqeqeq: an error should point to the equation symbol

### ISSUE THREAD

**Issue by mgol** [eslint/eslint#1029]
The file `a.js`:

``` js
/* global o */
if (o.a == o.b) {
   o.c = o.d;
}
```

gives the following result:

```
a.js
  2:4  error  Expected '===' and instead saw '=='  eqeqeq

✖ 1 problem
```

i.e. it points to the beginning of the whole expression containing the incorrect equality mark. It makes it more difficult to see what's wrong (imagine having something really long instead of `o.a`), it doesn't cooperate with IDE plugins that will mark a perfectly valid `o` token as wrong and it doesn't match jsHint which points to the end of the equation mark.
